docs/migrations/web-react/MIGRATION-v2.md

```diff
@@ -0,0 +1,305 @@
+# Migration Guide WIP
```

Comment:
Why WIP?

Reply:
Because I expect we will modify it before the actual release. It is not final.

Reply:
OK